### PR TITLE
Import files from private experiment

### PR DESCRIPTION
**Pull request by tresoldi**
I'm opening the PR so we can start discuss the code, no plans for immediate merge.

Bringing back the issues raised by @lingulist:

This is already very nice. Here are some notes:

- the feature set of clts is available through clts, no need to reproduce it in the code. It is in the CLTS package in form of a file features.json, and also part of the BIPA class, if you check this in the code in pyclts.
- I'd not add the bipa features to the rows. If the data doesn't have them, they are not needed, it is only that phoible has its own feature set. But adding a link to pyclts in form of the pyclts NAME would be useful and could also be done in phoible, in fact.
- If the original data comes in basic classes, like vowel and consonant, I'd in fact check this against the whole inventory and add these as "features" for each sound.
- To add sources, I'd try to do a very simple search for author and year and treat them all as a misc source, with first author, year, and the rest added as a string to "title".

### CODE REVIEW

File: cldfbench_eurasianinventories.py [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+import logging
+
+# Import MPI-SHH libraries
+from clldutils.path import git_describe
+from pycldf import StructureDataset
+from pyglottolog import Glottolog
+import pyclts
+
+# BIPA features for descriptors
+BIPA_FEATURES = [
```
Comment (LinguList) [inline cldfbench_eurasianinventories.py:20]: As I mentioned before: bipa features can be derived from clts directly, non need to add it here.

**Review by LinguList — APPROVED** [review]
looks nice to me. Apart from the bipa-dictionary of features, but that's a marginal issue.

File: cldfbench_eurasianinventories.py (hunk 1 above)
Comment (tresoldi) [reply cldfbench_eurasianinventories.py:20]: Is there a recommended way to access it? The most elegant way I found is to call `clts.bipa._feature_values.keys()`, which is better than having the list by myself, but accesses what is supposed to be a private member.

**Comment by LinguList** [comment]
Well, privacy is not really a feature of python, right? So I recommend,
just use that and not worry too much about it.

File: cldfbench_eurasianinventories.py (hunk 1 above)
Comment (xrotwang) [reply cldfbench_eurasianinventories.py:20]: @tresoldi  See https://github.com/cldf-clts/pyclts/issues/16 - and let me know whether you think the backwards compatible solution if better or not necessary.

Comment (tresoldi) [reply cldfbench_eurasianinventories.py:20]: @xrotwang perfect and elegant. :smile:

**Comment by xrotwang** [comment]
@tresoldi here's a tutorial on how to get started with using `cldfbench` "for real": https://github.com/cldf/cldfbench/blob/master/doc/tutorial.md
Should not be too hard, to back-port a skeleton as created with `cldfbench new` to your PR.

**Comment by tresoldi** [comment]
Thank you, @xrotwang !

**Comment by tresoldi** [comment]
Sorry, I unintentionally merged the PR. Will replace with the correct code right away.

**Pull request merged**